src/constants/popup.scss

```diff
@@ -1,7 +1,6 @@
 @import './styles';
 
 $popup-button-height: 38px;
-$popup-button-width: 38px;
```

Comment:
removed width because if there is a label associated with the popup button. it looks weird.